node/rpc/src/auth_api.rs

```diff
@@ -35,7 +38,9 @@ where
     B: Beacon + Send + Sync + 'static,
 {
     let ks = data.keystore.read().await;
-    let (token,) = params;
+    let (header_raw,) = params;
+    let token = header_raw.trim_start_matches("Bearer ");
+    info!("Auth verify token: {}", token);
```

Comment:
I'd remove these logs. They are already captured by the logs that print when we call an RPC method